atramhasis/templates/staticlayout.jinja2

```diff
@@ -6,6 +6,7 @@
 {% set app_name = app_name|default('Atramhasis') %}
 {% set copyright_year = copyright_year|default('2016-2017') %}
 
+{% block head %}
```

Comment:
Zou deze niet in `<head>` moeten staan?
Ik zou denken dat anders al de `<link>` urls na de `</head>` komen?

Reply:
Nee, daarmee kan ik alles tussen die `block` en `endblock` uitbreiden met `{{ super() }}`. Hij gaat in die templates niks invoegen.

Reply:
Ah ja, zo bedoel je. Het leek dat jinja slim genoeg was om dat dan tussen de tags te zetten (hij voegt nadien zelfs nog andere meta tags toe) maar ik heb het toch maar aangepast.